templates/shop/cart/complete/_google_pay.html.twig

```diff
@@ -0,0 +1,253 @@
+{% set payment = order.lastCartPayment() %}
+{% set gatewayConfig = payment.method.gatewayConfig %}
+{% set productionMode = gatewayConfig.config['production_mode'] ?? true %}
+{% set tpayMerchantId = gatewayConfig.config['merchant_id'] ?? '' %}
+{% set googleMerchantId = gatewayConfig.config['google_merchant_id'] ?? '' %}
+
+<div id="cw-tpay-google-pay-container" style="margin: 2em auto 0 auto; width: 330px; height: 54px;"></div>
+{{ form_row(form.tpay.google_pay_token) }}
+
+<script>
+    const ENVIRONMENT = '{{ productionMode ? 'PRODUCTION' : 'TEST' }}';
+    const TPAY_MERCHANT_ID = '{{ tpayMerchantId }}';
+    const GOOGLE_MERCHANT_ID = '{{ googleMerchantId }}';
+    const CURRENCY_CODE = 'PLN';
```

Comment:
We can get it right from the order, however we support only PLN currency. To be determined in https://github.com/CommerceWeavers/SyliusTpayPlugin/issues/73.